app/gui2/src/stores/graph/graphDatabase.ts

```diff
@@ -458,8 +464,10 @@ export interface Node {
   outerExprId: Ast.AstId
   pattern: Ast.Ast | undefined
   rootSpan: Ast.Ast
+  innerExpr: Ast.Ast
```

Comment:
We definitely need documentation here, describing what exactly is `outerExprId`, `root expression` and `inner expression`. And rename `rootSpan` to `rootExpr` for consistency.